etc/profile-m-z/strawberry.profile

```diff
@@ -0,0 +1,33 @@
+# Firejail profile for strawberry
+# Description: A music player and music collection organizer
+# This file is overwritten after every install/update
+# Persistent local customizations
+include strawberry.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/strawberry
+noblacklist ${HOME}/.config/strawberry
+noblacklist ${MUSIC}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+include whitelist-var-common.inc
+
+caps.drop all
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+# blacklisting of ioprio_set system calls breaks strawberry
+seccomp !ioprio_set
+
+private-dev
+private-tmp
```

Comment:
Some dbus-policies

Reply:
I don't know how to implement these in firejail, so I'm gonna need some more time for this too, reading from other profiles, finding out what strawberry needs, etc.

Reply:
If you are stuck it should be enough to block the system bus.
firejail >=0.9.63: `dbus-system none`
firejail <=0.9.62: `blacklist /run/dbus` (just for testing)

Reply:
I added `nodbus` to config files, is it ok?

Reply:
`nodbus` is deprecated. If it work with it add `dbus-{user,system} none`:

```
private-tmp

dbus-user none
dbus-system none
```

Reply:
I suppose not. Arch Linux firejail is 0.9.62 right now and its profiles use `nodbus` right now.
I will change it to `dbus-system none`.

Reply:
> I suppose not

We integrated xdg-dbus-proxy (see #3265 and #3406). 0.9.62 (the latest stable release) has of course still the old `nodbus`. Look at the profiles here at github.

Reply:
Actually I was answering myself:)

> - is it ok?
> - I suppose not, ... I will change it to dbus-system none.

You replied just before me.

Anyway nodbus has been removed, it changed to:
```
dbus-user none
dbus-system none
```

Reply:
You can configure shortcuts in strawberry to do some actions (play/pause, next track, previous track, etc.) and there is an option to "Use Gnome (GSD) D-Bus shortcut keys".

Also on track change, volume increase/decrease, etc. it shows a notification on gnome.

Enabling `nodbus` in profile breaks this functionality, as expected. I think disabling `dbus-user none` and `dbus-system none` would not be appropriate.

For notifications, I guess setting `dbus-user.talk org.freedesktop.Notifications` (taken from rhythmbox profile) would be enough, but I don't know how to allow gnome shortcut keys.

As I've said I'm on firejail 0.9.62 (I use arch btw:) and although I can install newer version (0.9.63) from git, I'm not sure I would do it most accurately and correctly, so does anyone have any opinion on this issue?

Reply:
If it is too much, you can leave it.

I wouldn't expect that `dbus-system none` breaks something. You can `blacklist /run/dbus` to get the same effect on 0.9.62.

Reply:
> I wouldn't expect that dbus-system none breaks something.

It breaks notifications and gnome-shortcut-keys as I've mentioned, isn't there a way to exclude these or in less secure way allow access to gnome dbus only?

Reply:
Really? If you have no `nodbus` and/or `dbus-user none`, only `dbus-system none`/`blacklist /run/dbus`.

>  gnome dbus 

There is no gnome-dbus, just the system-bus and the session-bus.
In firejail 0.9.63 you can use `dbus-user filter` with `dbus-user.{own,talk} NAME` for more control, but notifications should not be interact with the system-bus.

Reply:
I can't check it, becuase I'm on firejail 0.9.62, so if you say setting only `dbus-system none` will not cause gnome issues, I will remove `dbus-user none` leaving `dbus-system none` be. Am I right?

Reply:
`blacklist /run/dbus` has the same effect as `dbus-system none`

> Am I right?

yes